src/cli/lint/internalLints/upgrade/patches/13.1.1/patchRootTsconfig.test.ts

```diff
@@ -0,0 +1,191 @@
+import memfs, { vol } from 'memfs';
+
+import { configForPackageManager } from '../../../../../../utils/packageManager.js';
+import type { PatchConfig, PatchReturnType } from '../../index.js';
+
+import { patchRootConfig } from './patchRootTsconfig.js';
+
+jest.mock('fs-extra', () => memfs);
+jest.mock('fast-glob', () => ({
+  glob: (pat: string, opts: { ignore: string[] }) =>
+    jest.requireActual('fast-glob').glob(pat, { ...opts, fs: memfs }),
+}));
+jest.spyOn(console, 'debug').mockImplementation(() => {
+  /* empty */
+});
```

Comment:
What's this for?

Reply:
https://github.com/ast-grep/langs/issues/126 forgot to clean up